dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/src/Microsoft.ApplicationModel.Resources.vcxproj

```diff
@@ -154,6 +154,8 @@
       <OptimizeReferences>true</OptimizeReferences>
       <LinkIncremental>false</LinkIncremental>
       <GenerateDebugInformation>true</GenerateDebugInformation>
+      <!-- /PROFILE implies /OPT:REF /OPT:NOICF /INCREMENTAL:NO /FIXED:NO -->
+      <Profile>true</Profile>
```

Comment:
><Profile>true</Profile> [](http://example.com/codeflow?start=6&length=23)

Not relevant for API scanning, but I noticed this:
For this DLL, unlike mrm.dll, we're not setting `EnableCOMDATFolding`, `OptimizeReferences` and `GenerateDebugInformation` to `true` for Debug. Should we?

Reply:
This too is separate from your change. So feel free to ignore the comment.

Reply:
i think "<EnableCOMDATFolding>true</EnableCOMDATFolding>
      <OptimizeReferences>true</OptimizeReferences>" are wrong flags.

Reply:
I pushed additional changes with some more clean-up. Hopefully it makes things a bit clearer.

FWIW, I don't think we want to `EnableCOMDATFolding` for Debug builds, based on the comments on the page here:
https://docs.microsoft.com/en-us/cpp/build/reference/opt-optimizations?view=msvc-160

(Note: `/OPT:ICF` is `EnableCOMDATFolding=true`).

> /OPT:ICF can merge identical data or functions, it can change the function names that appear in stack traces. It can also make it impossible to set breakpoints in certain functions or to examine some data in the debugger, and can take you into unexpected functions when you single-step through your code. The behavior of the code is identical, but the debugger presentation can be very confusing. Therefore, we do not recommend that you use /OPT:ICF in debug builds unless the advantages of smaller code outweigh these disadvantages.